### PR TITLE
content/2023/kickstart-2023-wrap-up: Blog post (very old draft)

### PR DESCRIPTION
**Pull request by rkdarst**
- Draft post I found uncommitted in my working directory.  Did a bit
  of fix-up and PRing.
- Review: give it a good read and add more opinions to it.

### CODE REVIEW

File: content/2023/kickstart-2023-wrap-up.rst [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+
+
+
+General feedback
+----------------
+
+Our general feedback remains quite positive.  Our streaming +
+coteaching + collaborative notes format is still well received, and
+there seems to be little reason to go back for courses of smaller
+scale.  Reading + livestream + videos is a good combination.
```
Comment (ruokolt) [inline content/2023/kickstart-2023-wrap-up.rst]: What is this "reading" referring to? The collab notes, sphinx material?

File: content/2023/kickstart-2023-wrap-up.rst [hunk 2]
```diff
@@ -0,0 +1,156 @@
...
+Not enough time
+---------------
+
+There is never enough time - not much else to say.
+
+
+Reduce repetition
+-----------------
+
+Repitition is good, but not when it's a sign that we can't stop
```
Comment (ruokolt) [inline content/2023/kickstart-2023-wrap-up.rst]: Typo: Repetition

File: content/2023/kickstart-2023-wrap-up.rst [hunk 3]
```diff
@@ -0,0 +1,156 @@
...
+
+
+Kickstart 2023 wrap-up and thoughts for the future
+==================================================
+
+Our `kickstart course
+<https://scicomp.aalto.fi/training/scip/kickstart-2023/>`__ came and
+went with very few problems.  This post summarizes our general
+thoughts on the course and its format.
+
```
Comment (YuTian8328) [inline content/2023/kickstart-2023-wrap-up.rst]: Maybe shortly mention how the courses are evolving over years.

File: content/2023/kickstart-2023-wrap-up.rst [hunk 4]
```diff
@@ -0,0 +1,156 @@
...
+problem.
+
+Reflection: this will always be a problem in any course that has a
+wide enough audience.  We should accept and provide positive support
+for those not ready, and not try to exclude them.  It's OK to see a
+course and then strive to get the prerequisites later.
+
+
+
+Divide the course into two
```
Comment (ruokolt) [inline content/2023/kickstart-2023-wrap-up.rst]: The course was not divided so maybe this title should be "Should the course be divided into two?"

**Review by ruokolt — CHANGES_REQUESTED** [review]
Looks good! A few small comments here and there.

**Comment by rkdarst** [comment]
Various updates pushed.

File: content/2024/kickstart-2023-wrap-up.rst [hunk 5]
```diff
@@ -0,0 +1,178 @@
+:blogpost: true
+:date: 2024-02-13
```
Comment (rkdarst) [inline content/2024/kickstart-2023-wrap-up.rst]:
```suggestion
:date: 2024-02-15
```

**Pull request merged**